src/seedu/addressbook/ui/Formatter.java

```diff
@@ -0,0 +1,72 @@
+package seedu.addressbook.ui;
+
+import static seedu.addressbook.common.Messages.MESSAGE_GOODBYE;
+import static seedu.addressbook.common.Messages.MESSAGE_INIT_FAILED;
+import static seedu.addressbook.common.Messages.MESSAGE_PROGRAM_LAUNCH_ARGS_USAGE;
+import static seedu.addressbook.common.Messages.MESSAGE_USING_STORAGE_FILE;
+import static seedu.addressbook.common.Messages.MESSAGE_WELCOME;
+
+import java.util.List;
+import seedu.addressbook.commands.CommandResult;
+
+
+public class Formatter {
+	
+	private static final String LINE_PREFIX = "|| ";
+
+    /** A platform independent line separator. */
+    private static final String LS = System.lineSeparator();
+
+    private static final String DIVIDER = "===================================================";
+
+    /** Format of indexed list item */
+    private static final String MESSAGE_INDEXED_LIST_ITEM = "\t%1$d. %2$s";
+
+
+    /** Offset required to convert between 1-indexing and 0-indexing.  */
+    public static final int DISPLAYED_INDEX_OFFSET = 1;
+    
+	public Formatter() {
+		
+	}
+
+	public String[] formatWelcomeMessage(String version, String storageFilePath) {
+		String storageFileInfo = String.format(MESSAGE_USING_STORAGE_FILE, storageFilePath);
+		return new String[] {DIVIDER, DIVIDER, MESSAGE_WELCOME, version, MESSAGE_PROGRAM_LAUNCH_ARGS_USAGE, storageFileInfo,
+				DIVIDER};
+	}
+	
+	public String[] formatGoodByeMessage() {
+		return new String[] {MESSAGE_GOODBYE, DIVIDER, DIVIDER};
+	}
+	
+	public String[] formatInitFailedMessage() {
+		return new String[]{MESSAGE_INIT_FAILED, DIVIDER, DIVIDER};
+	}
+	
+	public String[] formatResultMessage(CommandResult result) {
+        return new String[]{result.feedbackToUser, DIVIDER};
+	}
+	
+	public String formatIndexedList(List<String> listItems) {
+		final StringBuilder formatted = new StringBuilder();
```

Comment:
can think of better, more indicative variable name than `formatted`, this leads to the question formatted What?